generator/generator.yml

```diff
@@ -362,6 +362,64 @@ modules:
       vrrpInstanceScriptMstrRxLowerPri:
         ignore: true # Non-metric display string.
 
+# Kemp Technologies LoadMaster
+#
+# https://kemptechnologies.com/files/packages/current/LM_mibs.zip
+# https://support.kemptechnologies.com/hc/en-us/articles/202375677-LoadMaster-SNMP-MIB-s
+  kemp_loadmaster:
+    walk:
+      - interfaces
+      - sysUpTime
+      - laNames
+      - laLoadInt
+      - ssCpuUser
+      - ssCpuSystem
+      - ssCpuIdle
+      - memory
+      - 1.3.6.1.4.1.12196.13.0 # VSdesc
+      - 1.3.6.1.4.1.12196.13.1 # VSentry
+      - 1.3.6.1.4.1.12196.13.2 # RSentry
+    lookups:
+      - source_indexes: [vSidx]
+        lookup: 1.3.6.1.4.1.12196.13.1.1.13 # vSName
+    overrides:
+      daemonState:
```

Comment:
Does this change over time? Usually things called "state" would be statesets, not info.

There's other states too below.

Reply:
I suppose the "state"'s won't change over time in the MIB's. Not entirely sure however. It seem like some of the "state"s has changed in the past. `vSState` for example, seem to have removed one of it's elements at some point (3 is missing);
```
vSState OBJECT-TYPE
  SYNTAX    INTEGER {
    inService(1),
    outOfService(2),
    disabled(4),
    sorry(5),
    redirect(6),
    errormsg(7),
    securityDown(8)
  }
```
`rSState` is similar to `vSState` in missing the 3:rd element.
`daemonState` & `haState` are simpler in that regard.

What is your recommendation?

Reply:
The "change over time" referrers to the device, not the MIB. Think about ethernet link state (changes over time when you plug/un-plug a cable) vs ethernet port type (gige, 10gige, sfp) that doesn't really change over time.

Reply:
Ah, I see. :)
In that case, they all may change on the device over time.